api/types/headlessauthn.go

```diff
@@ -45,3 +45,17 @@ func (h *HeadlessAuthentication) CheckAndSetDefaults() error {
 func (h *HeadlessAuthentication) setStaticFields() {
 	h.Kind = KindHeadlessAuthentication
 }
+
+// Stringify returns the readable string for a headless authentication state.
+func (h HeadlessAuthenticationState) Stringify() string {
```

Comment:
Any reason why this method is called `Stringify()` and not `String()`?

Reply:
Because the protobuf generated file creates a `String` method that returns strings like: "HEADLESS_AUTHENTICATION_STATE_APPROVED", so I wanted to make a method that returned something more user-facing.